docs/src/pages/nextjs/appdir.mdx

```diff
@@ -119,4 +119,54 @@ export default function Home() {
 }
 ```
 
+### Configure automatic hydration during SSR (recommended)
+
+> Added in `v5.6`
+
+By default, `useUploadThing` (and thus also the premade components) will fetch
+your server for the route endpoint permissions which will show a loading state.
+By utilizing server components, we can perform automatic hydration of this data
+instead to avoid the loading state and show the route permissions immediately.
+
+To add automatic SSR hydration, simply render the `<NextSSRPlugin />` hydration
```

Comment:
```suggestion
To add SSR hydration for all uploadthing components, simply render the `<NextSSRPlugin />` hydration
```

Reply:
i feel like it is a good thing to call out that this is an "all or nothing" feature that affects every uploadthing component (and doesn't need to be configured except at this global level)